recipes/pyflowy/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You should remove this file since it is included in the source distribution and conda/rattler will pick it up from there.